stagecoach/jobs/publishers/checkpoint/app.py

```diff
@@ -0,0 +1,50 @@
+import asyncio
+import json
+import os
+
+import boto3
+from empiric.core.logger import get_stream_logger
+from empiric.core.utils import str_to_felt
+from empiric.publisher.assets import EMPIRIC_ALL_ASSETS
+from empiric.publisher.client import EmpiricPublisherClient
+
+logger = get_stream_logger()
+
+
+def handler(event, context):
+    invocation = asyncio.run(_handler(EMPIRIC_ALL_ASSETS))
+    return {
+        "result": invocation,
+    }
+
+
+def _get_pvt_key():
+    secret_name = "publisherPrivateKey"
+    region_name = "us-west-1"
+
+    # Create a Secrets Manager client
+    session = boto3.session.Session()
+    client = session.client(service_name="secretsmanager", region_name=region_name)
+    get_secret_value_response = client.get_secret_value(SecretId=secret_name)
+    return int(
+        json.loads(get_secret_value_response["SecretString"])["PUBLISHER_PRIVATE_KEY"]
+    )
+
+
+async def _handler(assets):
+    private_key = _get_pvt_key()
+    account_address = int(os.environ.get("ACCOUNT_ADDRESS"))
+    pairs = [str_to_felt("/".join(p["pair"])) for p in assets if "pair" in p]
```

Comment:
Can you use `currency_pair_to_pair_id` from empiric/core/utils.py please instead

Reply:
done